### ISSUE TITLE
No yabridge vsts showing up on bitwig

### ISSUE THREAD

**Issue by kz-n** [robbert-vdh/yabridge#353]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

im trying to use yabridge with bitwig (Note: i'm new to both softwares, testing both) and i followed the install steps but none of my plugins are showing up in the devices list, theres also no mention of "wine" in the bitwig terminal output and trying to set the yabridge debug log env var to a file doesn't actually create said file, which leads me to think yabridge isnt working? i'm on endeavouros (arch based)

### Operating system

EndeavourOS 

### How did you install yabridge?

from yay, i think arch repos

### yabridge version

same as yabridgectl i guess i dont know

### yabridgectl version

yabridgectl 5.1.0

### Wine version

wine-8.19 ( TkG Staging Esync Fsync )

### Plugin type

None

### Plugin architecture

None

### Host

Bitwig 5.0.4

### Debug log

Debug log doesn't work

**Comment by cyber-leaf** [comment]
In works on my Ubuntu 24.04

**Comment by molove** [comment]
Stupid question, but have you set your paths correctly? Is Bitwig looking for plugins in the folder Yabridge is saving them to?

**Comment by robbert-vdh** [comment]
Yeah did you set up the plugin paths in Bitwig's settings (IIRC they should auto populate, but sometimes they don't) and are you using Bitwig from the AUR and not through Flatpak? The latter won't be able to work because it can't find Wine and/or yabridge.

**Comment by Yov00** [comment]
Have you checked in setting/plugins 'show crashed plugins'  they could be already found but crashed.

I am currently facing some 'cannot read metadata' problems with yabridge plugins

**Comment by razcore-rad** [comment]
I confirm that I'm also getting 'cannot read metadata' in Bitwig v5.3 Beta 2 with yabridgectl 5.1.1, wine-10.0-rc2 (Staging) (tried other wine versions too) on Ubuntu 24.10.

Plugins work fine in other DAWs or apps like Qtractor, MuSE, Ardour, Waveform 13, Carla. Don't see any other errors reported unfortunately:

```
[2024-12-19 09:54:02.118 float-library-metadata info] Starting plugin host process with command: /opt/bitwig-studio/bin/BitwigPluginHost-X64-SSE41 metadata 1234 7 1
PluginHost: Connecting to metadata service with id 7 on port 1234 using id 1
PluginHost: Metadata connection established
[2024-12-19 9:54:2.136 BITWIG_PLUGIN_HOST info] PluginHost: Reading VST 2.4 plugin metadata for /home/razcore-rad/.vst/yabridge/midiCurve.so
[2024-12-19 09:54:22.130 float-library-metadata error] Plugin host not responding reading metadata from file /home/razcore-rad/.vst/yabridge/midiCurve.so
[2024-12-19 09:54:22.136 notifications warning] midiCurve is taking a long time to scan: Please check that the plug-in does not have a dialog open that is obscured by Bitwig Studio's user interface
[2024-12-19 09:54:25.034 float-library-metadata info] Closing connection with plugin host: The user chose to abort after timeout
[2024-12-19 9:54:25.034 BITWIG_PLUGIN_HOST error] Plugin host metadata command reader error: 
[2024-12-19 09:54:25.037 indexer info] Could not read metadata for yabridge/midiCurve.so com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: Plugin host did not respond for 20.0 seconds
stdout:
[2024-12-19 9:54:2.136 BITWIG_PLUGIN_HOST info] PluginHost: Reading VST 2.4 plugin metadata for /home/razcore-rad/.vst/yabridge/midiCurve.so
[2024-12-19 9:54:25.034 BITWIG_PLUGIN_HOST error] Plugin host metadata command reader error: 
```

![image](https://github.com/user-attachments/assets/9e51b929-0cf8-4c8b-99c7-df3a029d7ef8)

**Comment by op-a-l** [comment]
###### Hi, 
I get the same issue about the plugins metadata. I use Bitwig 2.7 with flatpak, yabridgectl 5.1.1 and wine-9.22 (Staging) on **Arch** with Gnome.
![image](https://github.com/user-attachments/assets/5f8b6114-a429-4ac7-a99e-3b1affea18b6)

**Comment by simongehrig** [comment]
Same Issue here.
![Bildschirmfoto vom 2025-01-01 17-47-41](https://github.com/user-attachments/assets/5cfcad37-4637-4e7f-b8fa-49dce0023373)

**Comment by gaheldev** [comment]
Using the latest automated build worked for me: https://github.com/robbert-vdh/yabridge/actions/runs/13619791415

see [#395](https://github.com/robbert-vdh/yabridge/issues/395#issuecomment-2599791169)

**Comment by razcore-rad** [comment]
I confirm that the last artifact works for me. I'll upload them here cause they'll get deleted after some time

[yabridge-5.1.1-10-g61fa5c90.tar.gz](https://github.com/user-attachments/files/19287394/yabridge-5.1.1-10-g61fa5c90.tar.gz)
[yabridgectl-5.1.1-10-g61fa5c90.tar.gz](https://github.com/user-attachments/files/19287393/yabridgectl-5.1.1-10-g61fa5c90.tar.gz)
